<map id="django.contrib.auth.management.commands.createsuperuser.NotRunningInTTYException" name="django.contrib.auth.management.commands.createsuperuser.NotRunningInTTYException">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,79,432,119"/>
<area shape="rect" id="Node000002" title=" " alt="" coords="175,5,262,31"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="221,44,221,79,216,79,216,44"/>
</map>
